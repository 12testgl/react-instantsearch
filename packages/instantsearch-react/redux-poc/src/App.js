import React, { Component } from 'react';

import InstantSearch from './lib/InstantSearch';
import Hits from './lib/Hits';
import SearchBox from './lib/SearchBox';
import RefinementList from './lib/RefinementList';

class App extends Component {
  render() {
    return (
      <InstantSearch
        appId="latency"
        apiKey="6be0576ff61c053d5f9a3225e2a90f76"
        indexName="movies"
      >
        <div>
          <RefinementList attributeName="genre" />
          <SearchBox />
          <Hits />
        </div>
      </InstantSearch>
    );
  }
}

export default App;
