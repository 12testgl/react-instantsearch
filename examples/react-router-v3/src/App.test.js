import React from 'react';
import App from './App';
import { render } from '@testing-library/react';
import { Router, Route, browserHistory } from 'react-router';

describe('react-router-v3 recipe', () => {
  it('App renders without crashing', () => {
    const { container } = render(
      <Router history={browserHistory}>
        <Route path="/" component={App} />
      </Router>
    );

    expect(container).toMatchSnapshot();
  });
});
