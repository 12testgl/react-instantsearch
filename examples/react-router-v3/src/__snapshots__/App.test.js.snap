// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-router recipe App renders without crashing 1`] = `
<div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "justifyContent": "center",
        "marginBottom": 10,
      }
    }
  >
    <div
      className="ais-SearchBox"
    >
      <form
        action=""
        className="ais-SearchBox-form"
        noValidate={true}
        onReset={[Function]}
        onSubmit={[Function]}
        role="search"
      >
        <input
          autoCapitalize="off"
          autoComplete="off"
          autoCorrect="off"
          autoFocus={false}
          className="ais-SearchBox-input"
          maxLength="512"
          onChange={[Function]}
          placeholder="Search hereâ€¦"
          required={true}
          spellCheck="false"
          type="search"
          value=""
        />
        <button
          className="ais-SearchBox-submit"
          title="Submit your search query."
          type="submit"
        >
          <svg
            className="ais-SearchBox-submitIcon"
            height="10"
            viewBox="0 0 40 40"
            width="10"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M26.804 29.01c-2.832 2.34-6.465 3.746-10.426 3.746C7.333 32.756 0 25.424 0 16.378 0 7.333 7.333 0 16.378 0c9.046 0 16.378 7.333 16.378 16.378 0 3.96-1.406 7.594-3.746 10.426l10.534 10.534c.607.607.61 1.59-.004 2.202-.61.61-1.597.61-2.202.004L26.804 29.01zm-10.426.627c7.323 0 13.26-5.936 13.26-13.26 0-7.32-5.937-13.257-13.26-13.257C9.056 3.12 3.12 9.056 3.12 16.378c0 7.323 5.936 13.26 13.258 13.26z"
            />
          </svg>
        </button>
        <button
          className="ais-SearchBox-reset"
          hidden={true}
          title="Clear the search query."
          type="reset"
        >
          <svg
            className="ais-SearchBox-resetIcon"
            height="10"
            viewBox="0 0 20 20"
            width="10"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M8.114 10L.944 2.83 0 1.885 1.886 0l.943.943L10 8.113l7.17-7.17.944-.943L20 1.886l-.943.943-7.17 7.17 7.17 7.17.943.944L18.114 20l-.943-.943-7.17-7.17-7.17 7.17-.944.943L0 18.114l.943-.943L8.113 10z"
            />
          </svg>
        </button>
      </form>
    </div>
    <div
      className="ais-PoweredBy"
    >
      <span
        className="ais-PoweredBy-text"
      >
        Search by
      </span>
       
      <a
        aria-label="Algolia"
        className="ais-PoweredBy-link"
        href="https://www.algolia.com/?utm_source=react-instantsearch&utm_medium=website&utm_content=localhost&utm_campaign=poweredby"
        rel="noopener noreferrer"
        target="_blank"
      >
        <svg
          className="ais-PoweredBy-logo"
          viewBox="0 0 512 117"
        >
          <g
            clipPath="url(#a)"
            fill="#003DFF"
          >
            <path
              clipRule="evenodd"
              d="M249.538 64.18V1.379c0-.846-.756-1.49-1.59-1.36L236.184 1.87a1.378 1.378 0 0 0-1.163 1.361l.039 63.696c0 3.012 0 21.611 22.374 22.261a1.38 1.38 0 0 0 1.419-1.377v-9.507c0-.69-.51-1.285-1.193-1.362-8.125-.935-8.125-11.09-8.125-12.757l.003-.004Z"
              fillRule="evenodd"
            />
            <path
              d="M443.544 24.416h-11.839c-.759 0-1.375.616-1.375 1.375v62.036c0 .76.616 1.376 1.375 1.376h11.839c.759 0 1.375-.616 1.375-1.376V25.791c0-.76-.616-1.375-1.375-1.375Z"
            />
            <path
              clipRule="evenodd"
              d="M431.705 16.64h11.836c.76 0 1.376-.615 1.376-1.375V1.375c0-.843-.755-1.49-1.59-1.359L431.491 1.87a1.376 1.376 0 0 0-1.163 1.359v12.034c0 .76.615 1.376 1.375 1.376l.002.002ZM411.248 64.18V1.379c0-.846-.755-1.49-1.59-1.36L397.895 1.87a1.378 1.378 0 0 0-1.164 1.361l.04 63.696c0 3.012 0 21.611 22.373 22.261a1.38 1.38 0 0 0 1.42-1.377v-9.507c0-.69-.51-1.285-1.194-1.362-8.124-.935-8.124-11.09-8.124-12.757l.002-.004ZM380.456 33.272c-2.597-2.855-5.788-5.047-9.507-6.6-3.711-1.522-7.754-2.296-12.139-2.296-4.378 0-8.427.74-12.104 2.296-3.634 1.553-6.828 3.742-9.502 6.6-2.674 2.852-4.751 6.264-6.239 10.265-1.487 4.003-2.154 8.712-2.154 13.605 0 4.894.744 8.596 2.227 12.639 1.487 4.037 3.529 7.485 6.166 10.341 2.632 2.851 5.788 5.04 9.465 6.635 3.676 1.592 9.353 2.408 12.216 2.443 2.856 0 8.574-.89 12.286-2.443 3.711-1.557 6.868-3.784 9.507-6.635 2.632-2.856 4.674-6.304 6.12-10.341 1.45-4.043 2.154-7.745 2.154-12.639 0-4.893-.779-9.602-2.339-13.605-1.487-4.003-3.529-7.413-6.161-10.265h.004Zm-10.358 38.175c-2.674 3.672-6.425 5.526-11.211 5.526-4.791 0-8.539-1.819-11.213-5.526-2.674-3.671-4.012-7.928-4.012-14.267 0-6.269 1.303-11.456 3.977-15.123 2.669-3.672 6.42-5.483 11.206-5.483 4.791 0 8.54 1.813 11.211 5.483 2.674 3.63 4.048 8.854 4.048 15.123 0 6.339-1.334 10.56-4.008 14.267h.002ZM208.516 24.416h-11.5c-11.274 0-21.194 5.94-26.985 14.943-3.385 5.264-5.36 11.57-5.36 18.355 0 10.465 4.693 19.797 12.028 25.9.683.606 1.41 1.161 2.17 1.665 2.998 1.98 6.553 3.14 10.379 3.14.287 0 .573-.007.858-.02.084-.005.165-.013.249-.017.203-.012.408-.026.611-.047.079-.007.159-.018.238-.028.212-.023.424-.049.636-.079l.147-.023c7.668-1.182 14.352-7.185 16.529-14.643v13.482c0 .76.616 1.375 1.376 1.375h11.754c.76 0 1.375-.615 1.375-1.375V25.79c0-.76-.615-1.375-1.375-1.375h-13.13Zm0 48.239c-2.844 2.368-6.521 3.26-10.453 3.525-.038.002-.077.007-.115.01a12.48 12.48 0 0 1-.783.023c-9.847 0-17.979-8.368-17.979-18.504 0-2.39.457-4.665 1.264-6.756 2.615-6.789 9.038-11.596 16.566-11.596h11.5v33.298ZM497.492 24.416h-11.5c-11.274 0-21.194 5.94-26.985 14.943-3.385 5.264-5.359 11.57-5.359 18.355 0 10.465 4.692 19.797 12.027 25.9.683.606 1.41 1.161 2.17 1.665 2.998 1.98 6.553 3.14 10.379 3.14.287 0 .574-.007.858-.02.084-.005.166-.013.25-.017.202-.012.407-.026.61-.047.08-.007.159-.018.238-.028.212-.023.424-.049.637-.079l.146-.023c7.668-1.182 14.352-7.185 16.529-14.643v13.482c0 .76.616 1.375 1.376 1.375h11.754c.76 0 1.376-.615 1.376-1.375V25.79c0-.76-.616-1.375-1.376-1.375h-13.13Zm0 48.239c-2.844 2.368-6.52 3.26-10.453 3.525-.038.002-.077.007-.114.01-.262.016-.523.023-.784.023-9.847 0-17.979-8.368-17.979-18.504 0-2.39.457-4.665 1.264-6.756 2.616-6.789 9.038-11.596 16.566-11.596h11.5v33.298ZM306.344 24.416h-11.5c-11.274 0-21.194 5.94-26.985 14.943a33.74 33.74 0 0 0-5.154 14.592 34.617 34.617 0 0 0 .011 7.63c.998 8.88 5.395 16.695 11.811 22.036a19.46 19.46 0 0 0 2.17 1.664c2.998 1.98 6.554 3.14 10.379 3.14 4.194 0 8.069-1.382 11.228-3.723 3.797-2.7 6.733-6.653 8.038-11.132v11.718h-.026v2.583c0 5.091-1.331 8.921-4.042 11.507-2.707 2.583-7.236 3.877-13.58 3.877-2.592 0-6.712-.138-10.864-.562a1.382 1.382 0 0 0-1.461.984l-2.98 10.05a1.38 1.38 0 0 0 1.126 1.756c5.017.718 9.913 1.091 12.741 1.091 11.402 0 19.853-2.507 25.385-7.51 5.008-4.525 7.729-11.397 8.206-20.636V25.79c0-.76-.615-1.375-1.375-1.375h-13.13.002Zm0 14.943s.152 32.436 0 33.422c-2.816 2.277-6.32 3.168-10.138 3.427-.038.002-.077.007-.115.01-.261.015-.522.022-.783.022-.308 0-.613-.007-.918-.023-9.421-.492-17.373-8.686-17.373-18.506 0-2.39.457-4.665 1.263-6.756 2.616-6.789 9.039-11.596 16.566-11.596h11.501-.003Z"
              fillRule="evenodd"
            />
            <path
              d="M58.243 0C26.413 0 .466 25.665.007 57.387-.459 89.603 25.679 116.3 57.9 116.486c9.95.058 19.534-2.376 28.046-7.001.83-.45.958-1.592.252-2.217l-5.451-4.831a3.847 3.847 0 0 0-4.047-.68c-5.94 2.527-12.396 3.818-19.05 3.737-26.035-.32-47.07-21.982-46.656-48.016.41-25.705 21.448-46.488 47.249-46.488h47.253v83.99l-26.81-23.822c-.867-.771-2.196-.62-2.896.306-4.303 5.697-11.313 9.24-19.1 8.705-10.8-.746-19.552-9.442-20.361-20.238-.968-12.878 9.239-23.668 21.914-23.668 11.465 0 20.907 8.824 21.893 20.038a3.916 3.916 0 0 0 1.287 2.592l6.982 6.19c.793.702 2.05.273 2.245-.77.504-2.692.681-5.497.483-8.373C90.009 39.54 76.725 26.348 60.32 25.339 41.511 24.18 25.786 38.893 25.288 57.336 24.8 75.31 39.528 90.804 57.504 91.2a32.78 32.78 0 0 0 20.082-6.282l35.03 31.053c1.501 1.331 3.872.265 3.872-1.742V2.21A2.214 2.214 0 0 0 114.276 0H58.243Z"
            />
          </g>
          <defs>
            <clipPath
              id="a"
            >
              <path
                d="M0 0h512v116.565H0z"
                fill="#fff"
              />
            </clipPath>
          </defs>
        </svg>
      </a>
    </div>
  </div>
  <div
    style={
      Object {
        "display": "flex",
      }
    }
  >
    <div
      style={
        Object {
          "padding": "0px 20px",
        }
      }
    >
      <p>
        Hierarchical Menu
      </p>
      <div
        className="ais-HierarchicalMenu ais-HierarchicalMenu--noRefinement"
      />
      <p>
        Menu
      </p>
      <div
        className="ais-Menu ais-Menu--noRefinement"
      />
      <p>
        Refinement List
      </p>
      <div
        className="ais-RefinementList ais-RefinementList--noRefinement"
      />
      <p>
        Range Ratings
      </p>
      <div
        className="ais-RatingMenu ais-RatingMenu--noRefinement"
      >
        <svg
          style={
            Object {
              "display": "none",
            }
          }
          xmlns="http://www.w3.org/2000/svg"
        >
          <symbol
            id="ais-RatingMenu-starSymbol"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 .288l2.833 8.718h9.167l-7.417 5.389 2.833 8.718-7.416-5.388-7.417 5.388 2.833-8.718-7.416-5.389h9.167z"
            />
          </symbol>
          <symbol
            id="ais-RatingMenu-starEmptySymbol"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 6.76l1.379 4.246h4.465l-3.612 2.625 1.379 4.246-3.611-2.625-3.612 2.625 1.379-4.246-3.612-2.625h4.465l1.38-4.246zm0-6.472l-2.833 8.718h-9.167l7.416 5.389-2.833 8.718 7.417-5.388 7.416 5.388-2.833-8.718 7.417-5.389h-9.167l-2.833-8.718z"
            />
          </symbol>
        </svg>
        <ul
          className="ais-RatingMenu-list ais-RatingMenu-list--noRefinement"
        >
          <li
            className="ais-RatingMenu-item ais-RatingMenu-item--disabled"
          >
            <a
              aria-label="6 & Up"
              className="ais-RatingMenu-link"
            >
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <span
                aria-hidden="true"
                className="ais-RatingMenu-label"
              >
                 & Up
              </span>
               
              <span
                className="ais-RatingMenu-count"
              >
                0
              </span>
            </a>
          </li>
          <li
            className="ais-RatingMenu-item ais-RatingMenu-item--disabled"
          >
            <a
              aria-label="5 & Up"
              className="ais-RatingMenu-link"
            >
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <span
                aria-hidden="true"
                className="ais-RatingMenu-label"
              >
                 & Up
              </span>
               
              <span
                className="ais-RatingMenu-count"
              >
                0
              </span>
            </a>
          </li>
          <li
            className="ais-RatingMenu-item ais-RatingMenu-item--disabled"
          >
            <a
              aria-label="4 & Up"
              className="ais-RatingMenu-link"
            >
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <span
                aria-hidden="true"
                className="ais-RatingMenu-label"
              >
                 & Up
              </span>
               
              <span
                className="ais-RatingMenu-count"
              >
                0
              </span>
            </a>
          </li>
          <li
            className="ais-RatingMenu-item ais-RatingMenu-item--disabled"
          >
            <a
              aria-label="3 & Up"
              className="ais-RatingMenu-link"
            >
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <span
                aria-hidden="true"
                className="ais-RatingMenu-label"
              >
                 & Up
              </span>
               
              <span
                className="ais-RatingMenu-count"
              >
                0
              </span>
            </a>
          </li>
          <li
            className="ais-RatingMenu-item ais-RatingMenu-item--disabled"
          >
            <a
              aria-label="2 & Up"
              className="ais-RatingMenu-link"
            >
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <span
                aria-hidden="true"
                className="ais-RatingMenu-label"
              >
                 & Up
              </span>
               
              <span
                className="ais-RatingMenu-count"
              >
                0
              </span>
            </a>
          </li>
          <li
            className="ais-RatingMenu-item ais-RatingMenu-item--disabled"
          >
            <a
              aria-label="1 & Up"
              className="ais-RatingMenu-link"
            >
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starSymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <svg
                aria-hidden="true"
                className="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--empty"
                height="24"
                width="24"
              >
                <use
                  xlinkHref="#ais-RatingMenu-starEmptySymbol"
                />
              </svg>
               
              <span
                aria-hidden="true"
                className="ais-RatingMenu-label"
              >
                 & Up
              </span>
               
              <span
                className="ais-RatingMenu-count"
              >
                0
              </span>
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div
      style={
        Object {
          "display": "flex",
          "flex": 1,
          "flexDirection": "column",
        }
      }
    >
      <div
        style={
          Object {
            "display": "flex",
            "justifyContent": "space-around",
          }
        }
      >
        <div
          className="ais-ClearRefinements"
        >
          <button
            className="ais-ClearRefinements-button ais-ClearRefinements-button--disabled"
            disabled={true}
            onClick={[Function]}
          >
            Clear all filters
          </button>
        </div>
      </div>
      <div>
        <div
          className="ais-Hits"
        >
          <ul
            className="ais-Hits-list"
          />
        </div>
      </div>
      <div
        style={
          Object {
            "alignSelf": "center",
          }
        }
      />
    </div>
  </div>
</div>
`;
